sphinx/writers/latex.py

```diff
@@ -583,11 +583,18 @@ def visit_title(self, node):
             self.this_is_the_title = 0
             raise nodes.SkipNode
         elif isinstance(parent, nodes.section):
+            short = ''
+            if 'image' in str(node):
```

Comment:
This seems like a very brittle test.  Maybe replace it by `any(isinstance(n, nodes.image) for n in node.children)` or use `Node.traverse` or, not too bad as well, just do this always.